desktop/apps/auction/components/basic_checkbox/index.js

```diff
@@ -1,13 +1,13 @@
-import classNames from 'classnames'
+import PropTypes from 'prop-types'
 import React from 'react'
+import classNames from 'classnames'
```

Comment:
👍 no no it's good! I like when things are consistent, just wasn't sure how to deal with these cases (which is why my attempts were odd).